app/assets/stylesheets/views/_homepage.scss

```diff
@@ -251,3 +251,17 @@
     border-top: 4px solid $govuk-brand-colour;
   }
 }
+
+// Temporary overrides for global bar styling on homepage.
+// Doing this here avoids the need for republishing the
+// gem and therefore will be easier to remove later.
+.homepage.global-bar-present .global-bar {
+  border: 0;
+}
+
+.homepage.global-bar-present .homepage-header {
+  @include govuk-responsive-padding(7, "top");
+  @media (min-width: 1281px) { // stylelint-disable-line media-feature-range-notation
```

Comment:
Requested by Nikin to match similar tweaks for views beyond 1280px, see previous precedents in `_homepage_header.scss` for example [line 22](https://github.com/alphagov/frontend/blob/af433e2d0b937964023ee8870b0d8b926c6fe071/app/assets/stylesheets/views/_homepage_header.scss#L22).

Reply:
I've reviewed the design. Looks good on multiple viewport sizes 👍🏽

Reply:
I don't think in this case using the hard coded value is appropriate. The changes in `_homepage_header.scss` relate to making things 'extra big' on very large screens, whereas this is to do with how something appears on mobile. I don't want the use of a hard coded number to become a precedent here - we should be using the govuk-frontend breakpoints for consistency wherever possible.

Reply:
To my knowledge there are only [three breakpoints available](https://github.com/alphagov/govuk-frontend/blob/main/packages/govuk-frontend/src/govuk/settings/_media-queries.scss#L10), hence why this hardcoded 1281 value exists:
```
$govuk-breakpoints: (
  mobile:  320px,
  tablet:  641px,
  desktop: 769px
) !default;
```
Therefore I think it can remain and we look at refactoring this in another piece of work to add more named breakpoints.

Reply:
I disagree, for the reasons stated above. To clarify - the padding only needs to be increased on desktop, which we already have a breakpoint for in `govuk-frontend`. Why would this extra padding only need to be added above `1281px`?

Reply:
That's not the case, Nikin specifically asked for a little extra space on larger screens because the whole element looks unbalanced without it.

Reply:
Ah okay thanks that makes more sense now 👍

Reply:
I'm still uncomfortable about using magic numbers... might be something to feed back to the DS team on whether a 'large' breakpoint could be considered.

Reply:
I agree, we should avoid them where possible and I think it's probably time to revisit our defined breakpoints and how we use them.